token/cli/src/main.rs

```diff
@@ -270,13 +277,16 @@ pub fn signers_of(
     }
 }
 
+#[allow(clippy::too_many_arguments)]
 fn command_create_token(
     config: &Config,
     decimals: u8,
     token: Pubkey,
     authority: Pubkey,
     enable_freeze: bool,
     memo: Option<String>,
+    bulk_signers: Vec<Box<dyn Signer>>,
+    no_wait: bool,
```

Comment:
Sort of a nit, but the fact that no_wait is now passed to every command instruction is easy to misread as an indication that the no_wait functionality is available for various operations, when in truth it's only available for `command_transfer`.

Let's remove the line: `let mut no_wait = false;` and pass `matches.is_present("no_wait")` into `command_transfer` directly.